packages/cli/src/cmds/validator/options.ts

```diff
@@ -93,12 +94,18 @@ export const keymanagerOptions: CliCommandOptions<KeymanagerArgs> = {
     default: false,
     group: "keymanager",
   },
-  "keymanager.authEnabled": {
+  "keymanager.auth": {
```

Comment:
Updated the flag to enable authentication to be more inline with other existing boolean flags

Reply:
Probably worth documenting in next release, so that people can migrate to the preferred syntax?

Reply:
There is no need to migrate, existing setups can keep previous value